app/scripts/modules/amazon/src/securityGroup/details/securityGroupDetail.controller.js

```diff
@@ -167,7 +173,8 @@ module.exports = angular.module('spinnaker.amazon.securityGroup.details.controll
         account: securityGroup.accountId,
         applicationName: application.name,
         taskMonitorConfig: taskMonitor,
-        submitMethod: submitMethod
+        submitMethod: submitMethod,
+        retryBody: '<div><p>Retry deleting the security group and revoke any dependent ingress rules?</p><p>Any instance or load balancer associations will have to removed manually.</p></div>'
```

Comment:
Since we have fine-grained control over the error message, I wonder if it's worth trying to parse it to decide whether to show the retry message or not. Maybe it's added complexity and not worth it.